gulpfile.js

```diff
@@ -1,93 +1,59 @@
 import gulp from 'gulp';
 import fs from 'fs-extra';
 import axios from './bin/githubAxios.js';
-import minimist from 'minimist'
+import minimist from 'minimist';
 
 const argv = minimist(process.argv.slice(2));
 
-  gulp.task('default', async function(){
-  console.log('hello!');
-});
-
-const clear = gulp.task('clear', async function() {
-  await fs.emptyDir('./dist/')
-});
-
-const bower = gulp.task('bower', async function () {
-  const npm = JSON.parse(await fs.readFile('package.json'));
-  const bower = JSON.parse(await fs.readFile('bower.json'));
+// Task to clear the dist directory
+gulp.task('clear', async () => fs.emptyDir('./dist/'));
 
-  const fields = [
-    'name',
-    'description',
-    'version',
-    'homepage',
-    'license',
-    'keywords'
-  ];
-
-  for (let i = 0, l = fields.length; i < l; i++) {
-    const field = fields[i];
+// Task to update bower.json based on package.json
+gulp.task('bower', async () => {
+  const npm = await fs.readJson('package.json');
+  const bower = await fs.readJson('bower.json');
+  ['name', 'description', 'version', 'homepage', 'license', 'keywords'].forEach(field => {
     bower[field] = npm[field];
-  }
-
-  await fs.writeFile('bower.json', JSON.stringify(bower, null, 2));
+  });
+  await fs.writeJson('bower.json', bower, { spaces: 2 });
 });
 
+// Fetch contributors from GitHub
 async function getContributors(user, repo, maxCount = 1) {
-  const contributors = (await axios.get(
-    `https://api.github.com/repos/${encodeURIComponent(user)}/${encodeURIComponent(repo)}/contributors`,
-    { params: { per_page: maxCount } }
-  )).data;
-
-  return Promise.all(contributors.map(async (contributor)=> {
-    return {...contributor, ...(await axios.get(
-      `https://api.github.com/users/${encodeURIComponent(contributor.login)}`
-    )).data};
-  }))
+  const contributors = (await axios.get(`https://api.github.com/repos/${user}/${repo}/contributors`, { params: { per_page: maxCount } })).data;
+  return Promise.all(contributors.map(async contributor => ({
+    ...contributor,
+    ...(await axios.get(`https://api.github.com/users/${contributor.login}`)).data
+  })));
 }
 
-const packageJSON = gulp.task('package', async function () {
-  const CONTRIBUTION_THRESHOLD = 3;
```

Comment:
Do we really need to inline the constant? It feels like an unnecessary change, as constants are implemented to highlight configurable parameters for the task.